annotate.py

```diff
@@ -106,7 +106,10 @@ def annotate_img(img_path, size1, size2, initials) :
         if(k==ord('r')):
             if(pmtSelected == True):
                 print("Recording ", pmtID, "-",startingVal, " ",ix, "x  ", iy,"y\n")
-                file.write("%s\t%s-%d\t%d\t%d\t%s\n" %(filename,pmtID,startingVal, ix, iy,initials))
+                if(startingVal<10 or startingVal == 0):
+                    file.write("%s\t%s-0%d\t%d\t%d\t%s\n" %(filename,pmtID,startingVal, ix, iy,initials))
```

Comment:
[Here's a more programmatic way](https://stackoverflow.com/questions/339007/how-to-pad-zeroes-to-a-string) (non-hardcoding) to zero-pad a number.

Could you also pad the PMT ID? i.e. `%05d`

In general, try to find ways to avoid hardcoding numbers, reducing the number of `if` blocks (which the above method fixes), and instances of large copy/paste.

For example `"%s\t%s-%02d\t%d\t%d\t%s\n" %(filename,pmtID,startingVal, ix, iy,initials)` can be defined as one variable before, then used everywhere below.

Reply:
fixes #5

Reply:
just testing if this ^ works later when merging